README.md

```diff
@@ -10,7 +10,13 @@
     $ source ./venv/bin/activate
 
     $ pip install -r requirements.txt
-    
+
+    # Creating migration
+    $ python manage.py makemigrations
```

Comment:
When you change your models, you have to do makemigrations also
But in this case, it's redundant, because everybody who will clone your code have to have all migrations at migration folders